src/components/uncorrelatedEvents.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import EventFilter from 'components/Timeline/EventFilter'
+import Footer from 'components/Timeline/EventFooter'
+import Events from 'components/Timeline/Events'
+import { DateTime } from 'luxon'
+import * as filterActions from 'actions/filterActions'
+import TimePicker from 'material-ui/TimePicker'
+import DatePicker from 'material-ui/DatePicker'
+import { FlatButton } from 'material-ui'
+
+class uncorrelatedEvents extends Component {
```

Comment:
By convention, component class and function names should be capitalized. See: https://reactjs.org/docs/jsx-in-depth.html#specifying-the-react-element-type